app/services/proofing/lexis_nexis/ddp/verification_request.rb

```diff
@@ -0,0 +1,46 @@
+module Proofing
+  module LexisNexis
+    module Ddp
+      class VerificationRequest < Request
+        private
+
+        def build_request_body
+          {
+            api_key: config.api_key,
+            org_id: config.org_id,
+            account_address_street1: applicant[:address1],
+            account_address_street2: applicant[:address2] || '',
+            account_address_city: applicant[:city],
+            account_address_state: applicant[:state],
+            account_address_country: 'US',
+            account_address_zip: applicant[:zipcode].match(/^\d{5}/).to_s,
+            account_date_of_birth: applicant[:dob] ?
+              Date.parse(applicant[:dob]).strftime('%Y%m%d') : '',
+            account_email: applicant[:email],
+            account_first_name: applicant[:first_name],
+            account_last_name: applicant[:last_name],
+            account_telephone: '', # applicant[:phone], decision was made not to send phone
+            drivers_license_number_hash: applicant[:state_id_number] ?
+              OpenSSL::Digest::SHA256.hexdigest(applicant[:state_id_number].gsub(/\W/, '')) : '',
+            event_type: 'ACCOUNT_CREATION',
```

Comment:
is there a list of other event types? this is more like an account upgrade typically than an account create

Reply:
![Screen Shot 2022-08-15 at 2 36 08 PM](https://user-images.githubusercontent.com/34244063/184704956-eb74a633-faf7-48ff-a70c-21d2e64b3df5.png)

Reply:
seems to be the best option.  ie create a verified profile